components/Login/index.js

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { Form, Input, Button, Checkbox, Typography } from "antd";
+import { MailOutlined, LockOutlined } from "@ant-design/icons";
+import { useAuth } from "~/components/Auth";
+import Link from "next/link";
+
+import styles from "./style.module.css";
+
+function LoginForm() {
+  const { Title, Text } = Typography;
+  const { errors, isLoading, login } = useAuth();
+  // TODO: Session must be remembered when 'remember' is true
```

Comment:
Since we discussed really short times for the user authentication tokens, it really does not make a difference and the user has to explicitly logout since we don't even have access to them on the client.